cirq-ionq/cirq_ionq/service.py

```diff
@@ -110,7 +110,9 @@ def run(
             return result.to_cirq_result(params=cirq.ParamResolver(param_resolver))
         else:
             sim_result = cast(results.SimulatorResult, result)
+            # pylint: disable=unexpected-keyword-arg
             return sim_result.to_cirq_result(params=cirq.ParamResolver(param_resolver), seed=seed)
+            # pylint: enable=unexpected-keyword-arg
```

Comment:
why is this unexpected?

Reply:
AFAICT it's a false positive from pylint. I don't know how it tries to infer the type and figure out what args are allowed, and various things I tried didn't help. We could also disable this pylint check entirely since it's probably better handled by mypy (similar to the `not-callable` check).